integration/setup/common.go

```diff
@@ -334,7 +353,7 @@ func setupClient(tb testing.TB, ctx context.Context, uri string) *mongo.Client {
 }
 
 // startup initializes things that should be initialized only once.
-func startup() {
+func startup(tb testing.TB) {
```

Comment:
`startup` can't accept `testing.TB` because that TB would be from one random test, and that completely messes logging and other things